multiversion/src/main/scala/multiversion/commands/ExportCommand.scala

```diff
@@ -64,44 +68,48 @@ case class ExportCommand(
     parseThirdpartyConfig().flatMap(t => runResult(t))
   }
   def runResult(thirdparty: ThirdpartyConfig): Result[Unit] = {
-    withThreadPool[Result[Unit]] { threads =>
-      val coursierCache: FileCache[Task] = FileCache().noCredentials
-        .withCachePolicies(
-          List(
-            // first, use what's available locally
-            CachePolicy.LocalOnly,
-            // then, try to download what's missing
-            CachePolicy.Update
+    withThreadPool[Result[Unit]](
+      parallel,
+      { threads =>
+        val coursierCache: FileCache[Task] = FileCache().noCredentials
+          .withCachePolicies(
+            List(
+              // first, use what's available locally
+              CachePolicy.LocalOnly,
+              // then, try to download what's missing
+              CachePolicy.Update
+            )
           )
-        )
-        .withLocation(cache match {
-          case Some(c) => c.toFile
-          case _       => CacheDefaults.location
-        })
-        .withTtl(scala.concurrent.duration.Duration.Inf)
-        .withPool(threads.downloadPool)
-        .withChecksums(Nil)
-      for {
-        transformations <- thirdparty.transformations
-        resolutions <- initialResolutions(thirdparty, transformations, coursierCache)
-        index <- resolutionsWithAdditions(thirdparty, resolutions, transformations, coursierCache)
-        _ <- {
-          if (lint) lintPostResolution(index)
-          else ValueResult(())
-        }
-        output <- generateBzlFile(index, coursierCache)
-        _ = app.err.println(Docs.successMessage(s"Generated '$output'"))
-        lint <-
-          if (lint)
-            lintCommand
-              .copy(
-                queryExpressions = List("@maven//:all"),
-                app = app
-              )
-              .runResult()
-          else ValueResult(())
-      } yield lint
-    }
+          .withLocation(cache match {
+            case Some(c) => c.toFile
+            case _       => CacheDefaults.location
+          })
+          .withTtl(scala.concurrent.duration.Duration.Inf)
+          .withPool(threads.downloadPool)
+          .withChecksums(Nil)
+          .withRetry(retryCount)
```

Comment:
Will the retry apply to any kind of error? The doc of `--retry-count` suggests that it would apply only to Checksum errors, but I'm really not sure.

Reply:
I guess we can merge and find out without much risk.

Reply:
not sure about `retry`, there's no docstring for it, and it is also a string.

Reply:
also it was `resolve` specific, and `fetch` doesn't have it
```
$ ~/workspace/coursier/newer/cs  fetch --help  | rg retry
  --retry-count  <int>
```
I think it should be okay to merge.